src/main/app/src/Components/Manual/Man.tsx

```diff
@@ -16,105 +33,147 @@ interface Props {
  * Renders nivio manual, depending on which url param is given
  */
 const Man: React.FC<Props> = ({ setSidebarContent, setPageTitle }) => {
-  const [html, setHtml] = useState<string>("<p>This manual page doesn't exist. :(</p>");
+  const classes = useStyles();
+  const [html, setHtml] = useState<JSX.Element | JSX.Element[]>(
+    <React.Fragment>This manual page doesn't exist. :(</React.Fragment>
```

Comment:
Need to improve the response when there is no manual available.

- It will look more good if the "This manual Page doesn't exist " message appears  in the middle, centre aligned and  top of the page 
- And why we need a scroller if the content is just a single line? May be logic should be dynamically shown scroll bar when the page does have a manual content 
![image](https://user-images.githubusercontent.com/73687196/108592049-ed159980-7391-11eb-909e-29657d4de63f.png)

Reply:
Ok, please open a follow-up issue and we can merge this one. Thanks for the review.